RecoParticleFlow/PFClusterProducerAlpaka/plugins/BuildFile.xml

```diff
@@ -0,0 +1,35 @@
+<library name="RecoParticleFlowPFClusterProducersPlugins" file="*.cc">
+  <use name="FWCore/Framework"/>
+  <use name="FWCore/ParameterSet"/>
+  <use name="FWCore/Utilities"/>
+  <use name="alpaka"/>
```

Comment:
Is direct dependence on `alpaka` needed for the non-Alpaka-based plugins?

Reply:
No, it has been removed.